services/obs/obs.service.js

```diff
@@ -0,0 +1,94 @@
+import Joi from 'joi'
+import { BaseXmlService } from '../index.js'
+import { optionalUrl } from '../validators.js'
+import { isBuildStatus, renderBuildStatusBadge } from './obs-build-status.js'
+
+const schema = Joi.object({
+  status: Joi.object({
+    '@_code': isBuildStatus,
+  }).required(),
+}).required()
+
+export default class ObsService extends BaseXmlService {
+  static category = 'build'
+  static route = {
+    base: 'obs',
+    pattern: ':project/:packageName/:repository/:arch',
+    queryParamSchema: Joi.object({
+      instance: optionalUrl,
+    }).required(),
+  }
+
+  static auth = {
+    userKey: 'obs_userName',
+    passKey: 'obs_userPass',
+    authorizedOrigins: [],
+    isRequired: true,
+  }
+
+  static examples = [
+    {
+      title: 'openSUSE OBS package build status',
+      namedParams: {
+        project: 'openSUSE:Tools',
+        packageName: 'osc',
+        repository: 'Debian_11',
+        arch: 'x86_64',
+      },
+      staticPreview: this.render({
+        repository: 'Debian_11',
+        status: 'succeeded',
+      }),
+      keywords: ['open build service'],
+    },
+  ]
+
+  static defaultBadgeData = { label: 'OBS' }
```

Comment:
All our badge defaults need to adhere to our [specification](https://github.com/badges/shields/blob/master/spec/SPECIFICATION.md) which includes not using the service/platform name in the label

```suggestion
  static defaultBadgeData = { label: 'build' }
```